Doc/tutorial/datastructures.rst

```diff
@@ -93,7 +93,7 @@ objects:
 .. method:: list.copy()
    :noindex:
 
-   Return a shallow copy of the list.  Equivalent to ``a[:]``.
+   Return a shallow copy of the list.  Similar to ``a[:]``.
```

Comment:
IIRC, this one is really equivalent. And many scripts use `a[:]` instead of `.copy()`.